vendor/k8s.io/kubernetes/cmd/libs/go2idl/client-gen/main.go

```diff
@@ -182,7 +207,16 @@ func main() {
 			CmdArgs:             cmdArgs,
 		}
 	} else {
-		inputPath, groups, gvToPath, err := parseInputVersions()
+		var inputPath []string
+		var groups []types.GroupVersions
+		var gvToPath map[types.GroupVersion]string
+		var err error
+
+		if len(*inputVersions) == 1 && len(*groupFlag) > 0 {
```

Comment:
> In fact, I would prefer to just patch d83988e#diff-8141559b14bbf3947265d9b5fc691296R89 to extract our group name, or to add the group name somehow to the input list syntax.

I would just refactor out type packages to match kube formats, but that should be done post-rebase.